mender-update/daemon/state_test.cpp

```diff
@@ -486,7 +612,7 @@ vector<StateTransitionsTestCase> GenerateStateTransitionsTestCases() {
 					"ArtifactReboot_Error_00",
 					"ArtifactRollback_Enter_00",
 					"ArtifactRollback",
-					"ArtifactRollback_Leave_00",
+					"ArtifactRollback_Error_00",
```

Comment:
No error scripts on the error path, as described [here](https://github.com/mendersoftware/mender/pull/1368#discussion_r1305213570).